commercetools.Sdk/commercetools.Sdk.Linq/Query/Converters/CollectionPredicateVisitorConverter.cs

```diff
@@ -47,6 +48,22 @@ public IPredicateVisitor Convert(Expression expression, IPredicateVisitorFactory
             return new CollectionPredicateVisitor(predicateVisitors);
         }
 
+        private static bool IsStringOrEnum(object item, out string result)
+        {
+            switch (item)
+            {
+                case string stringValue:
+                    result = stringValue.WrapInQuotes();
+                    return true;
+                case Enum enumValue:
+                    result = enumValue.ToString().WrapInQuotes();
```

Comment:
it's better to use description enum attribute if exists to be like:
result = enumValue.GetDescription().WrapInQuotes();

Reply:
Fixed